example/index.html

```diff
@@ -0,0 +1,23 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Bitmovin Player Analytics Conviva Integration</title>
+    <link rel="stylesheet" href="./style.css">
+    <script src="https://cdn.bitmovin.com/player/web/8/bitmovinplayer.js"></script>
+    <script src="./conviva-core-sdk.min.js"></script>
+    <script src="../dist/bitmovinplayer-analytics-conviva.js"></script>
+  </head>
+  <body>
+  <div class="container">
+    <div class="content">
+      <div class="player-wrapper">
+        <div id="player"></div>
+      </div>
+    </div>
+  </div>
+  <script src="./script.js"></script>
```

Comment:
An example should be written with simplicity in mind and as easy to understand as possible. Extracting the JS code into a separate file introduces an unnecessary indirection for people trying to understand the code, especially inexperienced ones. I know this is a cleaner "architecture" but that's not the main focus of an example.